pom.xml

```diff
@@ -92,6 +92,16 @@
             <version>${swagger.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.24</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
I'm not sure that this dependency is actually needed.
Can we use directly the second one, with a fixed version?